Source/Core/Core/Boot/Boot.cpp

```diff
@@ -102,57 +177,24 @@ void CBoot::UpdateDebugger_MapLoaded()
   Host_NotifyMapLoaded();
 }
 
+// Get map file paths for the active title.
 bool CBoot::FindMapFile(std::string* existing_map_file, std::string* writable_map_file,
                         std::string* title_id)
```

Comment:
It's appropriate to remove the `title_id` parameter now that the debugger can get the same thing from `m_debugger_game_id`.